pkg/virtctl/imageupload/imageupload.go

```diff
@@ -171,7 +171,12 @@ func (c *command) run(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-
+	u, err := url.Parse(uploadProxyURL)
+	if err != nil {
+		return err
+	} else if u.Scheme == "" {
+		uploadProxyURL = fmt.Printf("http://$%s", uploadProxyURL)
```

Comment:
I guess that the `$` in `fmt.Printf` was not intentional?

Reply:
indeed :) fixed